staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -162,13 +166,16 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 		discovery: map[string]*discovery.APIGroupHandler{},
 		delegate:  delegateHandler,
 	}
+	establishingController := establish.NewEstablishingController(s.Informers.Apiextensions().InternalVersion().CustomResourceDefinitions(), crdClient.Apiextensions())
 	crdHandler := NewCustomResourceDefinitionHandler(
 		versionDiscoveryHandler,
 		groupDiscoveryHandler,
 		s.Informers.Apiextensions().InternalVersion().CustomResourceDefinitions(),
 		delegateHandler,
 		c.ExtraConfig.CRDRESTOptionsGetter,
 		c.GenericConfig.AdmissionControl,
+		establishingController,
+		c.ExtraConfig.MasterCount,
```

Comment:
this only applies when the `master-count` endpoint reconciler is used. that reconciler is no longer the default and is deprecated in 1.10 due to its inability to remove dead apiservers (https://github.com/kubernetes/kubernetes/pull/63383)

Reply:
> With the new lease type, is there a similarly simple way to detect HA?

@rphillips ^